Gemfile.lock

```diff
@@ -186,16 +186,16 @@ GEM
     has_scope (0.7.2)
       actionpack (>= 4.1)
       activesupport (>= 4.1)
-    hashdiff (1.0.0)
+    hashdiff (1.0.1)
     http-accept (1.7.0)
     http-cookie (1.0.3)
       domain_name (~> 0.5)
     i18n (0.9.5)
       concurrent-ruby (~> 1.0)
     ice_nine (0.11.2)
-    image_processing (1.10.0)
+    image_processing (1.10.3)
```

Comment:
### [image_processing](https://github.com/janko/image_processing)

**Patch** version upgrade :chart_with_upwards_trend::small_blue_diamond: 1.10.0 → 1.10.3

[_~~change-log~~, ~~source-code~~_]





<details>
<summary>Commits</summary>

A change of **9** commits. See the full changes on [the compare page](https://github.com/janko/image_processing/compare/v1.10.0...v1.10.3).

These are the individual commits:
- (26f15e2) [[vips] Simplify setting :Q](https://github.com/janko/image_processing/commit/26f15e222396bc86f8eebfbdb4587edef8affd77)
- (eda32f5) [Bold main documentation links](https://github.com/janko/image_processing/commit/eda32f5896e4bdd1cfa7a5115fc13876660d2a7e)
- (a9c5730) [[vips] Fix compatibility with ruby-vips 2.0.17+](https://github.com/janko/image_processing/commit/a9c57304c7139b8a39e9cf6a924272ca9e195f32)
- (799ae22) [Bump to 1.10.1](https://github.com/janko/image_processing/commit/799ae2218520d682edf0920f8331577d69b7c19d)
- (59ce62b) [Fix Ruby 2.7 warnings (#63)](https://github.com/janko/image_processing/commit/59ce62b703de84d131e95cb2aca7837cb150a090)
- (aac882c) [Bump to 1.10.2](https://github.com/janko/image_processing/commit/aac882c30998490f11ceb4ed216db5e326888de5)
- (5052cc5) [[vips] Fix auto-rotation bug on libvips 8.9.0](https://github.com/janko/image_processing/commit/5052cc552668b3ac2ee7f8b2470683afb5705044)
- (e7de626) [Improve accuracy of the gem description](https://github.com/janko/image_processing/commit/e7de626c93244759ee9eda63f05c3e994ab17f07)
- (69a87a9) [Bump to 1.10.3](https://github.com/janko/image_processing/commit/69a87a995fdde43ed539259e97e5bdce6806c67a)


</details>